jlm/hls/backend/rhls2firrtl/verilator-harness-hls.hpp

```diff
@@ -23,7 +23,41 @@ class VerilatorHarnessHLS : public BaseHLS
   std::string
   get_text(llvm::RvsdgModule & rm) override;
 
+public:
+
+  /**
+  * Construct a Verilator harness generator.
+  *
+  * /param verilogFile The filepath to the input Verilog file to Verilator
+  */
+  VerilatorHarnessHLS(const util::filepath verilogFile)
+    : VerilogFile_(verilogFile) {};
+
 private:
+  const util::filepath VerilogFile_;
+
+  /**
+   * Get the include name.
+   *
+   * \return The include name.
+   */
```

Comment:
The documentation is useless. It just states the method name. Which include name is returned? What is it used for?

Also, I would have returned a `util::filepath` here and not a string. You are always returning a valid file. The return type should reflect that.

Reply:
Changed to filepath and updated documentation.